Manito/Manito/Screens/Interaction/OpenManittoViewController.swift

```diff
@@ -198,16 +200,18 @@ final class OpenManittoViewController: BaseViewController {
 // MARK: - UICollectionViewDataSource
 extension OpenManittoViewController: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        guard let count = friendsList.count else { return 0 }
+        guard let count = self.friendsList.count else { return 0 }
         return count
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell: ManittoCollectionViewCell = collectionView.dequeueReusableCell(forIndexPath: indexPath)
-        if let colorIdx = friendsList.members?[indexPath.item].colorIdx {
+
+        if let colorIdx = self.friendsList.members?[indexPath.item].colorIdx {
```

Comment:
p2;

```suggestion
        if let colorIndex = self.friendsList.members?[indexPath.item].colorIdx {
```

여기도 약어로 되어 있네요.. 현재 `Member` 모델 내부에 idx로 되어있는데, 서버에서 받아오는 값이라 codingKey 처리가 필요할 것 같습니다!
같이 작업해주셔도 좋고, 추후에 수정해도 좋습니다! 현재 PR에선 Interaction 컨벤션 적용이니까요 ^________^

Reply:
[bcefa50](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/427/commits/bcefa501f34a6070508a5b781c4b3e1478ddaf39)

CodingKey 작업까지 해서 전체적으로 Index로 수정했습니다.